so_box_biogeo/diags/estNsq.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from MITgcmutils import rdmds, densjmd95
+from hspython import loadgrid
+import matplotlib.pyplot as plt
+"""
+    Estimating Brunt-Vaisala frequency using T and S
+    In case when N$^2$ is needed but do not have 'DRHODR' saved, 
+    one can still estimate it using T and S
+"""
+#
+# Constants
+#
+rhoconst = 1.035e3
+g = 9.81
+#
+# Load grid files
+# (hspython is available at https://github.com/hajsong/hspython)
+#
+grd = loadgrid('so_box', varname=['XC','YC','RC','hFacC','DRC','RF'])
+[nz, ny, nx] = grd.hFacC.shape
+#
+# Compute the stratification frequency using DRHODR. 
+# It is defined at the center of the layer.
+#
+dRHOdr = rdmds('ocestrat', 9, rec=0);    # DRHODR is in the first record in "ocestrat"
+Nsq = - dRHOdr*g/rhoconst*grd.mskC
+#
+# Now, estimate Nsq using T and S. 
+# When computing "drhodr" at the layer interface, density at upper and lower cell 
```

Comment:
Rather than "layer interface" it may be clearer to say the interface between tracer cells.